lib/hanami/hanamirc.rb

```diff
@@ -119,12 +118,21 @@ def exists?
 
     private
 
+    def parse(str)
+      {}.tap do |hash|
+        str.split("\n").each do |line|
```

Comment:
Instead of this split, we should change how `#content` works and rely on how Ruby iterates for each line of a file. See `File#readlines`.